docs/source/models/concept_models/vivarium_mncnh_portfolio/concept_model.rst

```diff
@@ -849,20 +849,27 @@ Default stratifications to all observers should include scenario and input draw.
     - 
     - 
   * - 7.0
+    - Wave I neonatal probiotics with scale-up scenarios 
+    - Baseline and alternative scenarios 2 - 10 
+    - ``model7.0``
+    - 
+    - 
+    - 
```

Comment:
If there are no desired changes to the default specifications let's start writing default to make that clear!